.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
 
 script:
   # Audit npm packages. Fail build whan a PR audit fails, otherwise report the vulnerability and proceed.
-  - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then npx audit-ci --low -a node-fetch ws; else npx audit-ci --low -a node-fetch ws || true; fi
+  - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then npx audit-ci --low -a node-fetch ws glob-parent yargs-parser; else npx audit-ci --low -a node-fetch ws glob-parent yargs-parser || true; fi
```

Comment:
Why is this change necessary?  Is it bypassing audit checks only for pull request checks or something?

Reply:
Could not make it pass, no safe version is available and another one requires major version library upgrade.

Reply:
@carrolp please help figure out how to deal with "ws" upgrade

Reply:
graphql-tools package (that depends on the bad version of ws) not actually used, and is now removed.